src/main.cpp

```diff
@@ -2758,7 +2759,7 @@ bool static DisconnectTip(CValidationState& state, const CChainParams& chainpara
             // ignore validation errors in resurrected transactions
             list<CTransaction> removed;
             CValidationState stateDummy;
-            if (tx.IsCoinBase() || !AcceptToMemoryPool(mempool, stateDummy, tx, false, NULL, true)) {
+            if (tx.IsCoinBase() || !AcceptToMemoryPool(mempool, stateDummy, tx, false, NULL, block.nTime, true, 0)) {
```

Comment:
What is the reasoning for using the reorg'd-out block timestamp as the accept time? Seems like it might defeat the purpose of resurrecting it in the first place?